caddytls/filestorage_test.go

```diff
@@ -0,0 +1,6 @@
+package caddytls
+
+// *********************************** NOTE ********************************
+// Due to circular package dependencies with the storagetest sub package and
+// the fact that we want to use that harness to test file storage, the tests
+// for file storage are done in the storagetest package.
```

Comment:
Possible option for stubbing out file operations: https://talks.golang.org/2012/10things.slide#8